CHANGELOG.md

```diff
@@ -6,10 +6,10 @@
 
 Timing diagram now includes events within each task/subworkflow and the rendered width is no longer hard-coded.
 
-### Added better error handling to authentication
-
 ### Increased minimum number of jobs loaded in the UI to better handle page size being set to 100.
 
+### Added better error handling to authentication
```

Comment:
I'd be more specific here and say "in cases where the API is trying to use oauth but the UI is not using CAAS"... or equivalent